evalai/utils/challenges.py

```diff
@@ -0,0 +1,64 @@
+import json
+import requests
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import urls
+from evalai.utils.common import valid_token
+
+
+DOMAIN = "http://localhost:8000"
+
+
+def get_challenges(url):
+
+    headers = get_headers()
+    response = requests.get(url, headers=headers)
+    response_json = json.loads(response.text)
+    if valid_token(response_json):
+
+        column_names = ['ID', 'Challenge Name', 'Short Description']
```

Comment:
I think we should remove the `Short description` from here as it can make the list too large to view when there will be a large number of challenges. 
@deshraj What do you say?

Reply:
@RishabhJain2018  @deshraj Yes, that's definitely a problem. The table just overflows into multiple lines and it becomes very ugly. I searched for substitute packages that solved this but I couldn't find a proper one, unfortunately.